frontend/src/metabase/visualizations/visualizations/PivotTable.jsx

```diff
@@ -168,7 +168,7 @@ export default class PivotTable extends Component {
       leftIndex.length === 0
         ? 0
         : LEFT_HEADER_LEFT_SPACING +
-          leftIndex[0].length * LEFT_HEADER_CELL_WIDTH;
+          leftIndex[0][0].length * LEFT_HEADER_CELL_WIDTH;
```

Comment:
This change is unrelated to the fixed heights but was another bug introduced in #14222. The `leftIndex` structure changed to accommodate including the subtotal rows.